scripts/west_commands/sign.py

```diff
@@ -12,6 +12,7 @@
 import shlex
```

Comment:
typo in commit: size, expanded by size off required additions, is compared agains
*against